contracts/exchange/contracts/test/TestValidatorWallet.sol

```diff
@@ -0,0 +1,296 @@
+/*
+
+  Copyright 2018 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.5;
+pragma experimental ABIEncoderV2;
+
+import "@0x/contracts-exchange-libs/contracts/src/LibOrder.sol";
+import "@0x/contracts-erc20/contracts/src/interfaces/IERC20Token.sol";
+import "@0x/contracts-utils/contracts/src/LibBytes.sol";
+
+
+interface ISimplifiedExchange {
+    function getOrderHash(LibOrder.Order calldata order)
+        external
+        view
+        returns (bytes32 orderHash);
+}
+
+
+// solhint-disable no-unused-vars
+contract TestValidatorWallet {
+    using LibBytes for bytes;
+
+    // Revert reason for `Revert` `ValidatorAction`.
+    string constant public REVERT_REASON = "you shall not pass";
+    // Magic bytes returned by EIP1271 wallets on success.
+    bytes4 constant public EIP1271_MAGIC_VALUE = 0x20c13b0b;
```

Comment:
I'm seeing this constant in a lot of places. Is there a single place it would make sense to keep this and inherit from?